.changeset/mighty-zoos-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": minor
+---
+
+feat(OnyxPagination): implement inline pagination
```

Comment:
In general, we should always write the changelog in the POV of the user what is new / changed for him :) So in this case, I'd prefer to say that we added a new property XYZ that allows to optionally display the pagination in an inline style